doc/source/examples/multiphysics/rayleigh-benard-convection/rayleigh-benard-convection.rst

```diff
@@ -0,0 +1,181 @@
+
+.. note::   
+    Note that the heating process is slow, and the velocity magnitudes are small inside the fluid. Hence, we expect large time-steps and a long simulation.
+
+.. code-block:: text
+
+    # --------------------------------------------------
+    # Simulation Control
+    #---------------------------------------------------
+    subsection simulation control
+        set method                          = bdf1
+        set time end                        = 10000
+        set time step                       = 0.01
+        set adapt                           = true
+        set max cfl                         = 0.5
+        set stop tolerance                  = 1e-5
+        set adaptative time step scaling    = 1.3
+        set number mesh adapt               = 0
+        set output name                     = rayleigh-benard_convection
+        set output control                  = time
+        set output time                     = 25
+        set output path                     = ./output/     
+    end
+
+
+The ``multiphysics`` subsection enables to turn on `(true)` and off `(false)` the physics of interest. Here ``heat transfer``, ``buoyancy force``, and ``fluid dynamics`` are chosen.
+
+.. code-block:: text
+
+    #---------------------------------------------------
+    # Multiphysics
+    #---------------------------------------------------
+    subsection multiphysics
+        set buoyancy force      = true
+        set heat transfer       = true
+        set fluid dynamics      = true
+    end 
+    
+The ``source term`` subsection defines gravitational acceleration. The value of the gravitational acceleration in this example is selected to satisfy the desired values of Ra number.
+
+.. code-block:: text
+    
+    #---------------------------------------------------
+    # Source term
+    #---------------------------------------------------
+    subsection source term
+        set enable                      = true
+        subsection xyz
+            set Function expression     = 0 ; -10 ; 0
+        end
+    end
+
+
+The ``physical properties`` subsection defines the physical properties of the fluid. Since we simulate the Rayleigh-Benard convection at two Rayleigh numbers (Ra=10000 and 25000), we use different thermal conductivities to reach mentioned Rayleigh numbers. We change the thermal conductivity of the fluid in the two simulations. Note that any other physical property (that is present in the Rayleigh number equation defined above) can be used instead of thermal conductivity. Both thermal conductivity values (:math:`k=0.15625` for :math:`Ra=10000`, and :math:`k=0.15625` for :math:`Ra=25000`) are added to the parameter handler file. However, only one of them should be uncommented for each simulation.
```

Comment:
```suggestion
The ``physical properties`` subsection defines the physical properties of the fluid. Since we simulate the Rayleigh-Benard convection at two Rayleigh numbers (Ra=10000 and 25000), we use different thermal conductivities to each mentioned Rayleigh numbers. We change the thermal conductivity of the fluid in the two simulations. Note that any other physical property (that is present in the Rayleigh number equation defined above) can be used instead of thermal conductivity. Both thermal conductivity values (:math:`k=0.15625` for :math:`Ra=10000`, and :math:`k=0.15625` for :math:`Ra=25000`) are added to the parameter handler file. However, only one of them should be uncommented for each simulation.
```

Reply:
Should there rather be 2 parameter files? I know it's only a line, but for other examples there were two parameter files when running for two different viscosities, for example.